triton-client/src/components/DatasetCard.tsx

```diff
@@ -242,29 +242,30 @@ function DatasetCard({ datasetID }: DatasetCardProps) {
 
     const showModal = useCallback(() => {
         Modal.info({
-            title: `Reads Per Sample of dataset #${datasetID}`,
+            title: `Reads Per Sample for run '${dataset?.run_name}' at lane ${dataset?.lane} for project '${project?.external_name}'`,
```

Comment:
"at lane..." does not sound right to me. Maybe "Reads Per Sample for lane X of run Y for project Z" ?